### DISCUSSION TITLE
Internal server error

### DISCUSSION THREAD

**Discussion by Alejandro-XIII** [rramosp/ai4eng.v1#139] [02 - PYTHON]
Buenas tardes, me sale este error en el laboratorio 02.04
![Captura](https://user-images.githubusercontent.com/78052009/173451133-e83c08a2-006e-4f03-8a92-48b6a0a05622.PNG)

En la página dice que el testing es correcto, pero la nota sigue en 0.0
![Captura](https://user-images.githubusercontent.com/78052009/173451472-230a514c-7051-4bfd-b745-2db13ec59b1d.PNG)

Ya intenté reiniciar el entorno y sigue igual, también intenté con un array y creo que no lo vale solo por ser array, porque lo veo bien ordenado.
![Captura](https://user-images.githubusercontent.com/78052009/173451809-595bf3d6-4089-4d7f-a1d7-623ba4ddea40.PNG)

**Comment by rramosp** [comment]
dale al botón de refrescar en tu pàgina del curso ... a veces recalcula de
nuevo tu nota para actualizar nuevas submisiones pero segùn los recursos
puede que tarde un poco

On Mon, Jun 13, 2022 at 4:56 PM Alejandro-XIII ***@***.***>
wrote:

> Buenas tardes, me sale este error en el laboratorio 02.04
> [image: Captura]
> <https://user-images.githubusercontent.com/78052009/173451133-e83c08a2-006e-4f03-8a92-48b6a0a05622.PNG>
>
> En la página dice que el testing es correcto, pero la nota sigue en 0.0
> [image: Captura]
> <https://user-images.githubusercontent.com/78052009/173451472-230a514c-7051-4bfd-b745-2db13ec59b1d.PNG>
>
> Ya intenté reiniciar el entorno y sigue igual, también intenté con un
> array y creo que no lo vale solo por ser array, porque lo veo bien ordenado.
> [image: Captura]
> <https://user-images.githubusercontent.com/78052009/173451809-595bf3d6-4089-4d7f-a1d7-623ba4ddea40.PNG>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/139>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIOVN4YOWW5JPFZBTODVO6U7TANCNFSM5YVT6A3Q>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Comment by Alejandro-XIII** [comment]
Listo profe, volví a mandar la respuesta y ya no me salió el error, muchas gracias
